### PR TITLE
fix-issue-443

### PR DESCRIPTION
**Pull request by qiaozha**
fix issue #443

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,6 @@ dist/
 .vscode/
 .vs/
 __pycache__/
-src/test/scenarios/testserver/input/
```
Comment (LianwMS) [inline .gitignore:6]: add this back

File: src/test/scenario-test.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ require('source-map-support').install();
         let msg = "";
         let finalResult = true;
         for (const each of folders) {
-            if (each != 'testserver') {
```
Comment (LianwMS) [inline src/test/scenario-test.ts:52]: > if (each != 'testserver') { [](start = 11, length = 28)

not run test server case in ci now.

Comment (qiaozha) [reply src/test/scenario-test.ts:52]: Just add some tests for your issue. and I don't think we can be sure this issue is fixed in the future if we don't add some baseline test. What if some other changes revert it back?

Comment (LianwMS) [reply src/test/scenario-test.ts:52]: As Kaihui are working on add more swagger for testserver, the baseline will be changed frequently currently. We will add it back once Kaihui complete the test server work. ETA is 1 month

---
In reply to: [448835496](https://github.com/Azure/autorest.az/pull/465#discussion_r448835496) [](ancestors = 448835496)

**Comment by qiaozha** [comment]
@LianwMS  please review again. Thx

**Pull request merged**